solarwinds/stackstate_checks/solarwinds/data/conf.yaml.example

```diff
@@ -0,0 +1,12 @@
+init_config:
+
+instances:
+  - url: [FQDN or IP address to SolarWinds Server]
+    instance_type: solarwinds
+    password: [API account password]
+    username: [API account username]
+    solarwinds_domain: [Custom property to check for STS inclusion]
+    solarwinds_domain_values:
+      - [Custom property value 1]
```

Comment:
is this specific to solarwinds?

Reply:
And this are Solarwinds domain values.
Yes, they are specific to SolarWinds.